tests/RuleSetTest.php

```diff
@@ -276,6 +266,21 @@ public function testSetDefinitionsAreSorted($setDefinitionName)
         ));
     }
 
+    /**
+     * @dataProvider provideSetDefinitionNameCases
+     *
+     * @param string $setDefinitionName
+     */
+    public function testSetDefinitionIsNotEmpty($setDefinitionName)
+    {
+        $reflection = new AccessibleObject(RuleSet::create());
```

Comment:
`new RuleSet()` ?

Reply:
Sure, done, any particular reason to change it here? I simply wonder why we even have this static constructor as it makes no difference comparing to normal creation.

Reply:
leftover (kept for bc reasons) of old php versions, where you could do `Foo::create()->bar();`, but you couldn't do it when using `new`